### DISCUSSION TITLE
IMPORTANT: Conversational language understanding & Composer

### DISCUSSION THREAD

**Discussion by jpcrodrigues** [microsoft/BotFramework-Composer#8963] [General]
According to Microsoft, CLU ([Conversational Language Understating](https://azure.microsoft.com/en-us/services/cognitive-services/conversational-language-understanding/#overview)) is the new next generation of LUIS.

So far as we know, Bot framework composer doesn't interoperate with CLU, do you have this integration in your road map? When can we expect this feature to be available on composer?

Hope you can answer as soon as possible.

**Comment by rschuetzler** [comment]
#9400
